lessons/specifics/poolboy.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: page
+title: Poolboy
+category: specifics
+order: 1
```

Comment:
@doomspork I like this idea. But then this lesson requires a category change instead of changing the order.
Should we create an issue about setting up the new category?